README.md

```diff
@@ -1,291 +1,177 @@
-
-- [Live Demo Link](https://google.com)
+> You can see a Demo here: [Demo](https://gdvl777.github.io/Leaderboard_ok/dist/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
+> To get a local copy up and running, follow these steps.
+> This project requires Node.js and npm installed on your machine.
 
-In order to run this project you need:
+-Node.js
+-npm
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
-
-### Install
+> -Clone this repository to your local machine using:
 
-Install this project with:
+>  git clone https://github.com/gdvl777/Leaderboard_ok.git
 
-<!--
-Example command:
+> -Navigate to the project folder:
 
-```sh
-  cd my-project
-  gem install
-```
---->
+> cd Leaderboard
 
-### Usage
+> -Install the project dependencies:
 
-To run the project, execute the following command:
+> npm install
 
-<!--
-Example command:
+> o start the development server, run the following command:
 
-```sh
-  rails server
-```
---->
+> npm start
 
-### Run tests
+### Project Structure
 
-To run tests, run the following command:
+> The project follows the following folder and file structure:
 
-<!--
-Example command:
+> /src: Contains the source files of the application.
+> /src/index.js: Main entry point of the JavaScript application.
+> /src/style.css: CSS file for the application styles.
+> /src/index.html: Base HTML file of the application.
+> /dist: Contains the generated production files.
+> webpack.config.js: Webpack configuration file.
 
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+### Setup
 
-### Deployment
+> Clone this repository to your desired folder: https://github.com/gdvl777/Leaderboard_ok.git
 
-You can deploy this project using:
+### Install
 
-<!--
-Example:
+> Install this project with: install WebLint and ESLint
 
-```sh
+### Usage
 
-```
- -->
+> To run the project, execute the following command: just need a web Browser
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Run tests
 
-<!-- AUTHORS -->
+> To run tests, run the following command: you just need a simple web browser to run this project for a test
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Claudia Rojas**
 
-👤 **Author2**
+-   GitHub: [@githubhandle](https://github.com/gdvl777)
+-   X: [LinkedIn](https://www.x.com/gdvl777_)
```

Comment:
- [ ] There is one name in the author and another name in the License. Please 🙏🏽 correct the Author section and make sure that the link to LinkedIn is the correct link to your LinkedIn profile.

![image](https://github.com/gdvl777/leaderboard_OK/assets/58642949/15ac0940-9768-4585-ba7b-8d4588e355e5)